app/views/dashboards/_dashboard_article_row.html.erb

```diff
@@ -31,6 +31,8 @@
   <div class="fs-s color-base-60">
     <% if !article.published? %>
```

Comment:
Reading note and personal preference: I am rarely a fan of `if !a … else … end` constructs.  They get a bit messy.

Reply:
An opportunity for the future improvement :eyes: